cmplibrary/src/release/java/com/sourcepoint/cmplibrary/creation/ComponentFactory.kt

```diff
@@ -12,11 +12,20 @@ import okhttp3.OkHttpClient
 
 internal fun getConnectionManager(appCtx: Context): ConnectionManager = ConnectionManager.create(appCtx)
 
-internal fun networkClient(appCtx: Context, netClient: OkHttpClient, responseManage: ResponseManager, logger: Logger): NetworkClient {
-    return createNetworkClient(
-        httpClient = netClient,
-        responseManager = responseManage,
-        urlManager = HttpUrlManagerSingleton,
-        logger = logger
-    )
-}
+internal fun networkClient(
```

Comment:
To be honest, I don't see much profit in having 3 copy-paste functions in
`cmplibrary/src/preprod/java/com/sourcepoint/cmplibrary/creation/ComponentFactory.kt`
and
`cmplibrary/src/release/java/com/sourcepoint/cmplibrary/creation/ComponentFactory.kt`
and
`cmplibrary/src/stage/java/com/sourcepoint/cmplibrary/creation/ComponentFactory.kt`
Is there any reason why we have it like this, instead of adding the function to a single base class?

Reply:
I agree with you. And they were very very annoying to work on. But I don't want to address them yet. It'll be part of a good refactoring after we are done integrating the `mobile-core`.